backend/src/apiserver/main.go

```diff
@@ -239,6 +296,50 @@ func startHttpProxy(resourceManager *resource.ResourceManager) {
 	glog.Info("Http Proxy started")
 }
 
+func startWebhook(controllerClient ctrlclient.Client) *http.Server {
```

Comment:
Sorry for going back on this, but now that I think of it more, it may be better to not have defaults on the webhookTLSCertPath and webhookTLSKeyPath CLI arguments and if they are set, then let `ListenAndServeTLS` fail. This is a security precaution that if the user intends to have TLS on by specifying the certs, then it should fail if the certs are not there rather than fallback to HTTP.

Reply:
@mprahl Since we considered this setup to enable seamless local testing, we may need to think of alternatives in this case.

Reply:
@VaniHaripriya I think it still works for local development with my comment because in that case, we just set empty values to the webhook TLS CLI arguments and the code should assume that means HTTP. I just want to error on the case where the user specifies the TLS CLI arguments and the paths don't exist.